spec/features/profile/profile_create_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.feature "Creating a profile" do
+  describe "after user registration" do
+    scenario "create his profile" do
+      visit new_user_registration_path
+      fill_in "user_email", with: "test@example.com"
+      fill_in "user_password", with: "password"
+      fill_in "user_password_confirmation", with: "password"
+      find("input[name='commit']").click
+      within("//div[@id='navMenu']") do
```

Comment:
We have changed the ids naming scheme.
Now instead of camel case, we use underscores 😉 

Change `within("//div[@id='navMenu']")` to `within("#nav_menu")` so as to adhere to the new naming convention.